Stephen-c-Kelly/life-choices-frontend
# Sk feat comments3

to test - 

- create post
- add comment
- see post, comment, and username of commenter on the page
- click a button to make a choice selection, observe that the color of the input changes
